docs/Bloom_Commands.md

```diff
@@ -34,6 +34,13 @@ filter entry.
     been exceeded. Performance will degrade linearly as the number of entries
     grow exponentially.
 
+Optional parameters:
+
+* **expansion**: When a new filter is created, its size will be the size of the
```

Comment:
Can you elaborate on the when?  I believe it's not really when it's created, but "when a threshold is reached" ... "an additional filter is created which size ...".